micro-rdk-ffi/src/ffi/runtime.rs

```diff
@@ -271,38 +273,23 @@ pub unsafe extern "C" fn viam_server_start(ctx: *mut viam_server_context) -> via
             ROBOT_SECRET.expect("Provided build-time configuration failed to set `ROBOT_SECRET`"),
         );
         log::info!("Robot configuration information was provided at build time - bypassing Viam provisioning flow");
-        #[cfg(target_os = "espidf")]
-        use micro_rdk::esp32::entry::serve_web_with_external_network;
-        #[cfg(not(target_os = "espidf"))]
-        use micro_rdk::native::entry::serve_web_with_external_network;
-        serve_web_with_external_network(None, repr, max_connection, ram_storage, network);
+        serve_with_network(None, repr, max_connection, ram_storage, network);
     }
 
     #[cfg(not(has_robot_config))]
     {
         #[cfg(not(target_os = "espidf"))]
-        {
-            use micro_rdk::common::credentials_storage::RAMStorage;
-            let storage = RAMStorage::default();
-            micro_rdk::native::entry::serve_web_with_external_network(
-                Some(ctx.provisioning_info),
-                repr,
-                max_connection,
-                storage,
-                network,
-            )
-        }
+        let storage = micro_rdk::common::credentials_storage::RAMStorage::default();
         #[cfg(target_os = "espidf")]
-        {
-            let storage = micro_rdk::esp32::nvs_storage::NVSStorage::new("nvs").unwrap();
-            micro_rdk::esp32::entry::serve_web_with_external_network(
-                Some(ctx.provisioning_info),
-                repr,
-                max_connection,
-                storage,
-                network,
-            )
-        }
+        let storage = micro_rdk::esp32::nvs_storage::NVSStorage::new("nvs").unwrap();
```

Comment:
nice :sunglasses:, didn't see the only difference was just the value of the storage passed in